presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.tests.hive;
+
+import com.teradata.tempto.ProductTest;
+import com.teradata.tempto.query.QueryExecutor;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.TestGroups.HIVE_CONNECTOR;
+import static com.facebook.presto.tests.TestGroups.QUARANTINE;
+import static com.facebook.presto.tests.utils.QueryExecutors.connectToPresto;
+import static com.teradata.tempto.assertions.QueryAssert.assertThat;
+import static java.lang.String.format;
+
+public class TestGrantRevoke
+    extends ProductTest
+{
+    /* This test has been quarantined since hive.security property is NOT set by default and
+     * doing so may affect other tests.
+     *
+     * Pre-requisites for the test:
+     * (1) In hive.properties file, set this property: hive.security=sql-standard;
+     * (2) tempto-configuration.yaml file should have two connections to Presto server:
+     * "prestoConfigUserAlice" that has "jdbc_user: alice" and
+     * "prestoConfigUserBob" that has "jdbc_user: bob"
+     * (all other values of the connection are same as the default "presto" connection).
+     * You can get these configurations from presto-product-tests/conf/tempto/tempto-configuration.yaml file.
+     *
+    */
```

Comment:
Do not use `quarantine` group for that. Please add separate group for that like `with_hive_auth_enabled`. Moreover if you need to have a special cluster configuration then you need to have your own separate docker images as the regular ones (like `cdh5-hive`) are used for regular tests. 

If I were you I would spent more time to make other tests to pass with your hive-site.xml. This would make all of this much easier to maintain and use. No need for special test group, no need for separate docker images, no special cases.